arm_robots/scripts/med_motion.py

```diff
@@ -5,47 +5,65 @@
+from victor_hardware_interface_msgs.msg import ControlMode
+from victor_hardware_interface.victor_utils import Stiffness
 
 def main():
     rospy.init_node('med_motion')
 
-    med = Med(manual_execute=True)
+    med = Med(display_goals=False)
     med.connect()
+    med.set_control_mode(ControlMode.JOINT_POSITION, vel=0.3)
     # med.set_grasping_force(40.0)
     # med.open_gripper()
 
-    med.plan_to_joint_config(med.arm_group, [0,1.0,0,1.0,0,1.0,0])
-    med.plan_to_joint_config(med.arm_group, [0,0,0,0,0,0,0])
-
+    med.plan_to_joint_config(med.arm_group, [0.0,1.0,0.0,1.0,0,1.0,0])
+    # input('Start impedance.')
+    # med.set_control_mode(ControlMode.JOINT_IMPEDANCE, vel=0.1, stiffness=Stiffness.SOFT)
+    # input('Go home.')
+    # med.set_control_mode(ControlMode.JOINT_POSITION, vel=0.1)
+    # med.plan_to_joint_config(med.arm_group, [0, 0, 0, 0, 0, 0, 0])
     exit()
 
+    # med.plan_to_joint_config(med.arm_group, [0, 1.0, 0, -1.0, 0, 1.0, 0])
+    # med.plan_to_pose(med.arm_group, med.wrist, [0.6, 0.12, 0.45, 0.0, np.pi, 0.0], frame_id='med_base')
+    # med.plan_to_position_cartesian(med.arm_group, med.wrist, target_position=[0.6, -0.12, 0.35])
+    # med.plan_to_position_cartesian(med.arm_group, med.wrist, target_position=[0.6, -0.12, 0.45])
+    # med.plan_to_position_cartesian(med.arm_group, med.wrist, target_position=[0.6, 0.12, 0.45])
+    # med.plan_to_position_cartesian(med.arm_group, med.wrist, target_position=[0.6, 0.12, 0.365])
+    # med.plan_to_position_cartesian(med.arm_group, med.wrist, target_position=[0.6, 0.12, 0.45])
+    # med.plan_to_joint_config(med.arm_group, [0, 0, 0, 0, 0, 0, 0])
+
+    # exit()
+
     # Add table plane.
     # scene = moveit_commander.PlanningSceneInterface(ns="victor")
     # scene.add_plane('table_plane', convert_to_pose_msg([0,0,0,0,0,0]))
 
     # Pick and place "demo"
-    med.plan_to_joint_config(med.arm_group, [0,0,0,0,0,0,0])
-
-    while True:
-        _, result, _ = med.plan_to_pose(med.arm_group, med.wrist, [0.6, 0.12, 0.45, 0.0, np.pi, 0.0], frame_id='med_base')
-        if result.error_code < 0:
-            replan = input('Replan? [y/n]')
-            if replan != 'y':
-                exit()
-        else:
-            break
-
-    med.plan_to_joint_config(med.arm_group, [0,0,0,0,0,0,0])
-    exit()
+    # med.plan_to_joint_config(med.arm_group, [0, 0, 0, 0, 0, 0, 0])
 
-    med.plan_to_position_cartesian(med.arm_group, med.wrist, target_position=[0.6, -0.12, 0.35])
-    med.grasp(15.0)
-    med.plan_to_position_cartesian(med.arm_group, med.wrist, target_position=[0.6, -0.12, 0.45])
-    med.plan_to_position_cartesian(med.arm_group, med.wrist, target_position=[0.6, 0.12, 0.45])
-    med.plan_to_position_cartesian(med.arm_group, med.wrist, target_position=[0.6, 0.12, 0.365])
-    med.release()
-    med.plan_to_position_cartesian(med.arm_group, med.wrist, target_position=[0.6, 0.12, 0.45])
-    med.plan_to_joint_config(med.arm_group, [0,0,0,0,0,0,0])
+    # while True:
```

Comment:
in general avoid commented out code, unless it would work if commented back in. It's good to keep stuff for reference/as exmaple but if it's just wrong/broken then delete it